src/tools/dart-devtools.md

```diff
@@ -5,13 +5,113 @@ description: A suite of debugging and performance tools.
 
 Dart DevTools is a suite of debugging and performance tools
 for Dart and Flutter.
+These tools are distributed in the [devtools package,][devtools package]
+**[PENDING: how do you normally get devtools, aside from pub global activate?]**
```

Comment:
`pub global activate` is one way to run the tool. It is also baked into `webdev` and the `Flutter` CLI.

Reply:
It sounds like there's no reason to `pub global activate devtools` if you're writing a web app. Correct?

Reply:
I think the most common way to run DevTools is from IDEs (VS Code and IntelliJ). The IDEs also support debugging web apps as well now.

Are we recommending web debugging through DevTools specifically, or documenting how to use from VS Code, IntelliJ, and DevTools?

Reply:
> It sounds like there's no reason to pub global activate devtools if you're writing a web app (and not a command-line app). Correct?

Correct.

> Are we recommending web debugging through DevTools specifically, or documenting how to use from VS Code, IntelliJ, and DevTools?

Yes. That's one of the main changes with this PR.